File: Labs/Lab-1/VP-1/src/vp/pkg1/Teacher.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package vp.pkg1;

/**
 *
 * @author Student
 */
public class Teacher extends Person{
    
    Teacher(){}
    
    public Teacher(String name, int age, long phoneNumber, String email) {
        this.setName(name);
        this.setAge(age);
        this.setPhoneNumber(phoneNumber);
        this.setEmail(email);
    }
    
    @Override
    public void behaviour() {
        System.out.println("Teaching!");
    }
}

File: Labs/Lab-2/Person/Main/Main/Teacher.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Main {
    class Teacher : Person {

        Teacher() { }

        public Teacher(String name, int age, Int64 phoneNumber, String email) {

            this.setName(name);
            this.setAge(age);
            this.setPhoneNumber(phoneNumber);
            this.setEmail(email);
        }

        public void behaviour() {
            Console.WriteLine("Teaching!");
        }

    }
}
